src/main/java/mods/railcraft/battery/SimpleBattery.java

```diff
@@ -6,7 +6,9 @@
  * Created by CovertJaguar on 1/15/2019 for Railcraft.
  *
  * @author CovertJaguar <http://www.railcraft.info>
+ * @deprecated Use the forge builtin module {@link net.minecraftforge.energy.IEnergyStorage IEnergyStorage}
  */
+@Deprecated
```

Comment:
Just remove, we're still indev so doesn't matter.

Reply:
they're still being used, going to swap them on next pr's